client/state/data-layer/wpcom/jetpack-install/index.js

```diff
@@ -47,8 +47,8 @@ export const handleError = ( action, error ) => {
 	const { retryCount = 0 } = dataLayer;
 
 	const logToTracks = withAnalytics(
-		recordTracksEvent( 'calypso_jpc_remote_install_api_response', {
-			remote_site_url: url,
+		recordTracksEvent( 'calypso_jpc_remote_install_api_failure', {
+			url: url,
 			data: JSON.stringify( error ),
```

Comment:
The error includes the request body in the error, let's make sure even bad credentials don't land in tracks:

```json
{
  "path": "/jetpack-install/…",
  "method": "POST",
  "apiVersion": "1.1",
  "query": "",
  "body": {
    "user": "demo",
    "password": "bad"
  },
  "callback": "tvh0y62",
  "supports_args": true,
  "supports_error_obj": true,
  "supports_progress": true,
  "name": "LoginFailureError",
  "statusCode": 400,
  "status": 400,
  "message": "",
  "error": "LOGIN_FAILURE"
}
```

Reply:
Wow, nice spot :boom:

Let me sort that.